apps/ehr/src/pages/Appointments.tsx

```diff
@@ -223,7 +223,7 @@ export default function Appointments(): ReactElement {
 
       if (
         (locationID || locationSelected?.id || providers.length > 0 || groups.length > 0) &&
-        (searchDate || appointmentDate) &&
+        searchDate &&
         Array.isArray(visitType)
```

Comment:
similarly here, we're forcing `visitType` to be an array (even if empty), so this conditional should be removed. so [line 232](https://github.com/masslight/ottehr/pull/2780/files#diff-51460d88a7ba925b5c627b42c601528f6a8339406267620330ffd83a9785028aL232) could be simplified to `visitType,`